src/psyclone/tests/psyad/utils_test.py

```diff
@@ -73,3 +75,28 @@ def test_active_passive(fortran_reader):
     assert not node_is_active(assignment, [])
     assert node_is_passive(assignment, [symbol_c])
     assert not node_is_active(assignment, [symbol_c])
+
+
+def test_negate_expr(fortran_writer):
+    '''Test that the negate_expr function negates a PSyIR expression by
+    multiplying it by minus one.
+
+    '''
+    # positive literal value
+    literal = Literal("1", INTEGER_TYPE)
+    result = negate_expr(literal)
+    assert fortran_writer(result) == "-1"
+    # negative literal value
+    literal = Literal("-1", INTEGER_TYPE)
+    result = negate_expr(literal)
+    assert fortran_writer(result) == "1"
+    # unary minus
+    minus = UnaryOperation.create(
+        UnaryOperation.Operator.MINUS, Literal("1", INTEGER_TYPE))
```

Comment:
Please add a test for e.g. UnaryOperation.Operator.PLUS as well.

Reply:
Done => -1 * (+1) !